workspaces/src/error/mod.rs

```diff
@@ -19,7 +19,7 @@ pub enum ErrorKind {
     #[error("Execution")]
     Execution,
     /// An error having to do with running sandbox.
-    #[error("Sandbox({0})")]
+    #[error("{0}")]
```

Comment:
Note: `error` is Display, it should not use the `TypeName(contents)` form, that one is only for Debug.